### PR TITLE
update docs + a few small changes

### PR DESCRIPTION
**Pull request by sean1588**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,25 @@ resources:
...
+## Outputs
+
+- originURL - the direct URL of the website (storage bucket endpoint)
+- cdnURL - the CDN URL for the site
+- customDomainURL - the custom domain URL where the static website can be accessed
+
+
+## Notes:
+
+- The SSL certs can take anywhere from 60 - 90 mins after the update has been completed in order to fully provision. Upon first provisioning, there may be a short period of time where the cert is invalid when accessing the website over HTTPS.
```
Comment (cnunciato) [inline README.md:52]: Good callout. I remember this taking a while.

File: provider/cmd/pulumi-resource-google-cloud-static-website/website.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export class Website extends pulumi.ComponentResource {
             }, { parent: this });
         });
 
-        this.originURL = pulumi.interpolate`https://storage.googleapis.com/${bucket.name}/index.html`;
+        this.originURL = pulumi.interpolate`https://storage.googleapis.com/${bucket.name}/${this.indexDocument || "index.html"}`;
```
Comment (cnunciato) [inline provider/cmd/pulumi-resource-google-cloud-static-website/website.ts]: Is this necessary, given the default on line 45?

**Review by cnunciato — APPROVED** [review]
Thanks for getting this over the line!

File: provider/cmd/pulumi-resource-google-cloud-static-website/website.ts (hunk 2 above)
Comment (sean1588) [reply provider/cmd/pulumi-resource-google-cloud-static-website/website.ts]: ahh, yes correct. it is already being set. thanks.

**Pull request merged**